src/amp-story-player/amp-story-player-impl.js

```diff
@@ -756,16 +770,27 @@ export class AmpStoryPlayer {
    * Shows the story provided by the URL in the player and go to the page if provided.
    * @param {?string} storyUrl
    * @param {string=} pageId
+   * @param {{animate: boolean?}} options
    * @return {!Promise}
    */
-  show(storyUrl, pageId = null) {
+  show(storyUrl, pageId = null, options = {}) {
     const story = this.getStoryFromUrl_(storyUrl);
 
     let renderPromise = Promise.resolve();
     if (story.idx !== this.currentIdx_) {
       this.currentIdx_ = story.idx;
 
       renderPromise = this.render_();
+
+      if (options?.animate != undefined) {
```

Comment:
no need for optional chaining here, we can keep it simple

```suggestion
      if (options.animate === false) {
```